restapi/config.go

```diff
@@ -38,6 +38,10 @@ var TLSPort = "8443"
 // TLSRedirect m3 tls redirect rule
 var TLSRedirect = "off"
 
+// defaultTenantMemorySize default value used
+// when generating minioInstance request
+var defaultTenantMemorySize = "4Gi"
```

Comment:
I think lets move this to 16Gi and also storage to 1Ti by default we should choose sane deployment defaults, no one will deploy us with 1Gi disk.

Reply:
I'll change the capacity to 16Gi then, Storage is an input from the user.

Reply:
done @harshavardhana